about.html

```diff
@@ -0,0 +1,126 @@
+            <li><a href="">Events</a></li>
+            <li><a href="">Sponsor</a></li>
+            <li><a href="">News</a></li>
+            <li><a href="" class="Fbutton">Gallery</a></li>
+          </ul>
+      </div>
+    </nav>
+  </header>
+  <body>
+    <section class="aboutme" id="aboutme_sec">
+      <div class="aboutme-content">
+        <p class="second-heading">"Hello Art Gallery"</p>
+        <h2 class="aboutme-heading">CREATIVE COMMONS<br>GLOBAL SUMMIT<br>2022</h2>
+        <div class="aboutme-text">
+          <p class="paragraph2">
+            A fun festival where people from more than 80 countries who believe in the value of sharing and openness and create positive change gather and share their thoughts will be held in Korea in October.
+          </p>
+        </div>
+        <p class="contact">Please contact us per Email for any further questions about CC Summit 2022!</p>
+        <p class="email">ccsummit2022@cckorea.org</p>
+      </div>
+      <div class="aboutme-info">
+        <h3 class="aboutme-paragraph">
+          Creative Commons Global Summit 2022 Logo
+        </h3>
+        <hr class="aboutme-line">
+        <p>The logo of CC Global Summit 2022 was decided through the logo competition from 7 march to 12 march.</p>
+        <div class="Image-logo">
+          <img src="./images/logo-first.png " alt="" width="300" height="100">
+        </div>
+      </div>
+      <hr>
+      <div class="Previous-events">
+        <h3>See the past Capablanca's Memorial</h3>
+        <hr class="event-line">
+        <p>Take a look at the last CC Global Summits which took place in Buenos Aires and in Warsaw.</p>
+        <div class="event-cards">
+          <div class="event-card">
+            <img src="./images/event2020.jpg" alt=""width="300" height="200">
+            <p class="year">2019</p>
+            <p class="city">Capablanca's Memorial in Buenos Aires</p>
+          </div>
+          <div class="event-card">
+            <img src="./images/event2021.jpg" alt="" width="300" height="200">
+            <p class="year">2020</p>
+            <p class="city">Capablanca's Memorial in Warsaw</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="partner">
+      <div class="partnerM">
+        <h3>Partners</h3>
+        <div class="p-container">
+          <div class="partner-item1">
+            
+              <img src="./images/mozilla.png" alt="" width="100" height="80">
+              <img src="./images/google6.png" alt="" width="100" height="90">
+              <img src="./images/Firefox.png" alt="" width="70" height="50">
+            </div>
+      
+            <div class="partner-item2">
+              
+                <img src="./images/humb.png" alt="" width="150" height="100">
+                <img src="./images/airbnb.png" alt="" width="100" height="50">
+              
+            </div>
+          
+          </div>
+        </div>
+    </section>
```

Comment:
- [ ] First, in the desktop view, the `about` page does not have a partner section, it only has the dark `footer` but yours has a dark `partner` section, a white space then a dark `footer` section. Kindly correct this.
- Your desktop design:
![Annotation 2022-03-11 063234](https://user-images.githubusercontent.com/85113567/157809476-a75ad579-3d4f-481a-951b-6aa801611bc2.jpg)
- Required desktop design:
![Annotation 2022-03-11 063256](https://user-images.githubusercontent.com/85113567/157809518-2e07f5a5-d5a2-4694-8a73-394081a450e8.jpg)
- [ ] Next, in the mobile view of the about page, it has a dark `partner` section with a white `footer` but yours has a dark `partner` section, a white space then a dark `footer`. Do well to correct this too so as to follow the design guidelines 👍 
- Your mobile view:
![Annotation 2022-03-11 063150](https://user-images.githubusercontent.com/85113567/157809828-78ef06df-30f9-49fc-a00f-9305cacd3496.jpg)
- Required mobile view:
![Annotation 2022-03-11 063210](https://user-images.githubusercontent.com/85113567/157809865-588debeb-2f04-48ca-9251-17785a6a5b1e.jpg)

Reply:
Have arranged it.